packages/utils/src/validator/isMap/index.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @description 주어진 값이 `Map` 타입인지 확인합니다. `Map` 타입이 맞다면 인자의 타입은 `Map`으로 타입을 좁혀줍니다.
+ *
+ * @template K - `Map` 요소 `key값`의 타입을 지정하는 제네릭 타입. 기본값은 `unknown`입니다.
+ * @template V - `Map` 요소 `value값`의 타입을 지정하는 제네릭 타입. 기본값은 `unknown`입니다.
+ *
+ * @param {unknown} value - `Map`인지 확인할 값.
+ * @returns {value is Map<K, V>} - 값이 `Map`이면 `true`, 그렇지 않으면 `false`를 반환합니다.
+ *
+ * @example
+ * const maybeMap = new Map([['hello', 5], ['world', 5]]);
+ * if (isMap<string, number>(maybeMap)) {
+ *   maybeMap; // 이제 Map으로 안전하게 사용될 수 있습니다.
+ * }
+ *
+ * const notMap = { hello: 5, world: 5 };
+ * isMap(notMap); // false
+ */
+export function isMap<K, V>(value: unknown): value is Map<K, V> {
+  return value instanceof Map;
+}
```

Comment:
```suggestion
export function isMap(value: unknown): value is Map<any, any> {
  return value instanceof Map;
}
```
```ts
const value = new Map() as Map<string, number>;

if (isMap(value)) {
  value; // const value: Map<string, number>
} else {
  value;
}
```
@zzzRYT 제네릭은 불필요합니다. 어차피 인자의 타입에 따라 타입 추론이 정상적으로 동작합니다 ㅎㅎ 제네릭으로 인한 타입 복잡도를 줄이면 좋을 것 같습니다 -> 문서, 주석 수정

Reply:
@ssi02014 리뷰 감사합니다.
저는 타입을 강제시켜서, 안정성을 올리는게 좋다고 생각했습니다. 
`isMap`은 단순히 `Map` 인지 아닌지를 판단하는 함수라, 크게 문제될 건 없어 보이네요. 금방 수정해서 올리겠습니다!

Reply:
@zzzRYT 네 만약에 최악의 경우라면 아래와 같은 현상도 발생 할 수 있을 것 같아요
isMap에다 타입을 지정하기보다 그냥 map 자체 타입을 자연스럽게 추론시키는게 좋을 것 같습니다 😄
```ts
const map = new Map<string, number>();

if (isMap<number, string>(map)) {
  map; // const map: Map<string, number> & Map<number, string>
}
```